### ISSUE TITLE
Validate name, email, and password credentials provided by user for registration

### ISSUE THREAD

**Issue by jkwening** [TheDevPath/googleMaps-offline-navigator#74]
User authorization and authentication is setup but currently we're not validating the user information provided for registering an account. The `registerUser` function in "users-controller.js" needs to be expanded to include this validation step first before any other processing steps are started.

Here's an overview of what is needed:
```
  /**
   * TODO
   * 1. validate to ensure null values are not passed for name,
   *   email, or password.
   * 2. verify email is valid i.e. <username>@<domain>
   * 3. validate for secure passwords and no easy to guess passwords
   */
```

**Comment by vettyvignesh** [comment]
Hi I am new to OSS. So can you assign me this issue?

**Comment by tanyagupta** [comment]
@jkwening this issue is still showing as help wanted. Any remaining issues still available?

**Comment by jkwening** [comment]
@tanyagupta @vettyvignesh is working on this. At the issues main page there should be other issues unassigned. Feel free to review them and comment that you're working on them and I or @motosharpley will assign it to you. There are frontend, backend, test/QA, and design issues that we need help on - even if not noted as "help wanted" as long as its not assigned feel free to volunteer for it and just let us know.

**Comment by jkwening** [comment]
Merged in #87
